tests/test_serialization.py

```diff
@@ -95,6 +95,25 @@ def test_serialize(laplace, model, reg_loader):
     assert torch.allclose(f_var, f_var2)
 
 
+@pytest.mark.parametrize('laplace', set(flavors_no_llla) - {LowRankLaplace, KronLaplace, FullLaplace})
```

Comment:
Why is this is test only run for `DiagLaplace`?

Reply:
It's an outdated commit. It's fixed in the latest one.